.golangci.yml

```diff
@@ -0,0 +1,10 @@
+linters:
+  enable:
+    - errcheck
+    - gosimple
+    - govet
+    - ineffassign
+    - staticcheck
+    - unused
+    - gofumpt
+    - bodyclose
```

Comment:
A few more that I think would be nice to add, but need some more changes so can be in a followup pr:

* `nakedret` (configured to not allow them at all, IMO)
* `predeclared`
* `unconvert` (may be controversial--perhaps a redundant convert is done for clarity)

I really want to use `goheader`, but it messes up docs: https://github.com/denis-tingaikin/go-header/issues/35.